runtime/caml/misc.h

```diff
@@ -120,19 +120,30 @@ CAMLdeprecated_typedef(addr, char *);
   #define Caml_noinline
 #endif
 
-/* Export control (to mark primitives and to handle Windows DLL) */
-
-#ifndef CAMLDLLIMPORT
-  #if defined(SUPPORT_DYNAMIC_LINKING) && defined(ARCH_SIXTYFOUR) \
-      && (defined(__CYGWIN__) || defined(__MINGW32__))
-    #define CAMLDLLIMPORT __declspec(dllimport)
+/* Export control (to mark primitives and to handle Windows DLLs and
+ * ELF visibility) */
+#if (defined(__CYGWIN__) || defined(__MINGW32__))
```

Comment:
Does someone know why this is restricted to Cygwin and MinGW-w64? MSVC obviously supports `__declspec`.